### PR TITLE
Remove ReplicaSet Nested Resource

### PR DESCRIPTION
**Pull request by jihochang**
ReplicaSet is not a public feature and there are no customers using it. It was an internal prototype that we are now removing (behind a feature flag).

I have reviewed the ReplicaSets change with Gaurav Bhatnagar, Chris Eggert, and Anthony Martin.

Let me know if you have any questions.

### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by jihochang** [comment]
Adding WaitForARMFeedback #7064

File: specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/domainservices.json [hunk 1]
```diff
@@ -311,260 +308,21 @@
...
-        "x-ms-long-running-operation": true
-      }
     }
   },
   "definitions": {
     "Resource": {
       "description": "The Resource model definition.",
       "properties": {
         "id": {
-          "readOnly": true,
```
Comment (raych1) [inline specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/domainservices.json]: From ARM guidance, these properties have to be `readonly`. Please see CI check failures for details.

**Review by raych1 — CHANGES_REQUESTED** [review]
Removing API from existed version is breaking change.

**Comment by anthony-c-martin** [comment]
> Removing API from existed version is breaking change.

Note that this feature has always been in private preview - from an ARM perspective, I'm fine with making this change.

**Review by anthony-c-martin — APPROVED** [review]
Signing off for ARM - assuming you'll be addressing https://github.com/Azure/azure-rest-api-specs/pull/7627#discussion_r338855162.

**Comment by jihochang** [comment]
Fixed the readonly property issue.

File: specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/domainservices.json (hunk 1 above)
Comment (jihochang) [reply specification/domainservices/resource-manager/Microsoft.AAD/stable/2017-06-01/domainservices.json]: This has been fixed.

**Comment by jihochang** [comment]
@raych1 Could you complete the merge?

**Review by raych1 — APPROVED** [review]

**Pull request merged**